client/index.html

```diff
@@ -265,6 +265,38 @@ <h1 class="title">Settings</h1>
 									Enable notification for all messages
 								</label>
 							</div>
+							<% if (!public) { %>
+							<div id="change-password">
+								<form action="" method="post">
+									<div class="col-sm-12">
+										<h2>Change password</h2>
+									</div>
+									<div class="col-sm-12">
+										<label aria-label="Enter current password">
+											<span class="sr-only">Enter current password</span>
+										</label>
+										<input class="input" type="password" name="old_password" placeholder="Enter current password">
```

Comment:
It appears, after discussing with an accessibility expert, that I got it wrong: even though `label`s and `aria-label` serve the same purpose, `label`s are the preferred way to do so.

Therefore, you can fix your code with these steps:
- Remove `aria-label`
- Move `class="sr-only"` to the `label`s themselves
- Remove the now empty `span`s